monday/query_joins.py

```diff
@@ -167,6 +166,75 @@ def delete_item_query(item_id):
     return query
 
 
+# COLUMNS RESOURCE QUERIES
+def create_column(
+    board_id: int, column_title: str, column_type: None, defaults: dict[str, any] = None, description: str = None
```

Comment:
`column_type` typed as None is likely erroneous. `ColumnTypes` seems to be from doc string below